.github/CODEOWNERS

```diff
@@ -2,38 +2,39 @@
 # Order is important. The last matching pattern takes the most precedence.
 
 # Default owners for everything in the repo.
-*        @elixirschool/developers
+*        @elixirschool/english
 
-en/      @elixirschool/english
+/lessons/en/      @elixirschool/english
```

Comment:
since it was `en/` before, should the new path be `lessons/en/` or do both work?

Reply:
I _think_ we just need the latter since the former no longer exists. Thoughts?

Reply:
Sorry for the confusion, I was just referring to the leading slash 😅 

So:

`/lessons/en` vs
`lessons/en`

Reply:
Oh! Yeah, that makes sense

Reply:
I'm not sure if it makes a difference tbh so if you want to merge this and then check that works!